### PR TITLE
[replace_bridge_nick] Add replace_bridge_nick.pl

### PR DESCRIPTION
**Pull request by mrskug**
Script to reformat the output from irc bridge bots to make chat more readable.

In short it turns this,
```
14:00 <+bridgebot> <herp> hello
14:00 <+bridgebot> <derp> hi
14:01 <+bridgebot> <derpina> morning
```

into this,
```
14:00 <+herp>  hello
14:00 <+derp>  hi
14:01 <+derpina>  morning
```

### CODE REVIEW

File: scripts/replace_bridge_nick.pl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+sub msg_strip_bridge_nick {
+    my ($tdest, $data, $stripped) = @_;
+    my $server = $tdest->{server};
+    my $bridge_nick = Irssi::settings_get_str('bridge_botnick');
+    return unless $server;
+
+    if ( $stripped =~ /^.?.?.?\Q$bridge_nick\E.?.?\<[^>]+\>/s ){
+        $stripped =~ /(^.?.?.?\Q$bridge_nick\E.?.?)(\<([^>]+)\>)/s;
+        my ($a,$b,$c) = ($1,$2,$3);
```
Comment (vague666) [inline scripts/replace_bridge_nick.pl]: don't use $a and $b, these are special to perl

File: scripts/replace_bridge_nick.pl [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#
+
+sub msg_strip_bridge_nick {
+    my ($tdest, $data, $stripped) = @_;
+    my $server = $tdest->{server};
+    my $bridge_nick = Irssi::settings_get_str('bridge_botnick');
+    return unless $server;
+
+    if ( $stripped =~ /^.?.?.?\Q$bridge_nick\E.?.?\<[^>]+\>/s ){
+        $stripped =~ /(^.?.?.?\Q$bridge_nick\E.?.?)(\<([^>]+)\>)/s;
```
Comment (vague666) [inline scripts/replace_bridge_nick.pl]: you can incorporate line 28 into line 27, no need to run the matching twice

**Comment by vague666** [comment]
I'm not sure about this script
The current method, as seen below, is more involved so I don't know if rewriting the output this way in the script has some drawbacks
`/trigger add -rawin -regexp ':[^!]*!~wesdiscor@(wesnoth/bot/discord-bridge) PRIVMSG #(wesnoth.*) :<([][[:alpha:]_{}^`|][][[:alnum:]_{}^`|\-]*)> (.*)' -replace ':$3|d!~wesdiscor@$1 PRIVMSG #$2 :$4'`

**Comment by ailin-nemui** [comment]
@mrskug see if you can implement the suggestions by vague666

**Comment by ailin-nemui** [comment]
closing for inactivity

**Pull request closed without merging**